### ISSUE TITLE
Changelog out of date

### ISSUE THREAD

**Issue by unsystemizer** [HelloZeroNet/ZeroNet#910]
It was created (https://github.com/HelloZeroNet/ZeroNet/issues/606) but hasn't been maintained.

I used to watch commits, but since there's been many translation updates it's hard to follow changes that way now. Can changelog be updated and kept up to date?

**Comment by HelloZeroNet** [comment]
Yeah, Thanks, I forgot about that. Updated: https://github.com/HelloZeroNet/ZeroNet/blob/master/CHANGELOG.md
